TheStandardBox.Core/Attributes/Contollers/GeneratedControllerAttribute.cs

```diff
@@ -16,58 +17,17 @@ public class GeneratedControllerAttribute : Attribute
     {
         public GeneratedControllerAttribute(
             string route,
-            AllowedAction[] allowedActions = null,
-            AllowedAction[] anonymousActions = null)
+            Action disabledMethods = Action.None,
```

Comment:
In this case you can disable only one action?!
For example can I disable `PutEntity` and `PostEntity` at the same time?

Reply:
We can disable them as Flags
anonymousActions = AllowedAction.PostEntity | AllowedAction.PutEntity